server/src/main/java/org/spine3/server/event/enrich/EventEnrichmentsMap.java

```diff
@@ -45,33 +46,43 @@
      */
     private static final String PROPS_FILE_PATH = "enrichments.properties";
 
-    private static final ImmutableMap<TypeName, TypeName> enrichmentsMap = buildEnrichmentsMap();
+    /**
+     * A separator between event types in the `.properties` file.
+     */
+    private static final String EVENT_TYPE_SEPARATOR = ",";
+
+    private static final ImmutableMultimap<TypeName, TypeName> enrichmentsMap = buildEnrichmentsMap();
 
     private EventEnrichmentsMap() {}
 
     /**
      * Returns the immutable map instance.
      */
     @SuppressWarnings("ReturnOfCollectionOrArrayField") // is immutable
-    /* package */ static ImmutableMap<TypeName, TypeName> getInstance() {
+    /* package */ static ImmutableMultimap<TypeName, TypeName> getInstance() {
         return enrichmentsMap;
     }
 
-    private static ImmutableMap<TypeName, TypeName> buildEnrichmentsMap() {
-        final ImmutableMap.Builder<TypeName, TypeName> builder = ImmutableMap.builder();
+    private static ImmutableMultimap<TypeName, TypeName> buildEnrichmentsMap() {
+        final ImmutableMultimap.Builder<TypeName, TypeName> builder = ImmutableMultimap.builder();
         final ImmutableSet<Properties> propertiesSet = loadAllProperties(PROPS_FILE_PATH);
         for (Properties properties : propertiesSet) {
             putTo(builder, properties);
         }
-        final ImmutableMap<TypeName, TypeName> result = builder.build();
+        final ImmutableMultimap<TypeName, TypeName> result = builder.build();
         return result;
     }
 
-    private static void putTo(ImmutableMap.Builder<TypeName, TypeName> enrichmentsMap, Properties properties) {
+    @SuppressWarnings("MethodWithMultipleLoops") // is OK in this case
```

Comment:
Please have a method object that would perform operations from `buildEnrichmentsMap()` and `putTo()`.